fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSignature.java

```diff
@@ -113,4 +118,34 @@ public FunctionSignature varArgs(AbstractDataType...argTypes) {
             return FunctionSignature.of(returnType, true, argTypes);
         }
     }
+
+    public static class MultiFuncSigBuilder {
```

Comment:
static number initialized when class loading. It has mulit-thread problem. So i don't think use this singleton object to build multi signature is a good idea.